src/main/java/org/jboss/xavier/analytics/functions/HelperFunctions.java

```diff
@@ -1,9 +1,55 @@
 package org.jboss.xavier.analytics.functions;
 
+import java.util.Arrays;
+
 public class HelperFunctions
 {
     public static int round(double value)
     {
         return (int) Math.round(value);
     }
+
+    public static boolean isSupportedOS(String osToCheck)
+    {
+        return Arrays.stream(OSSupport.values()).anyMatch(value -> value.name().equalsIgnoreCase(osToCheck) && value.isSupported());
```

Comment:
We need to change from "equalsIgnoreCase" approach to "startsWtith" approach having the values both lower case
```suggestion
        return Arrays.stream(OSSupport.values()).anyMatch(value -> value.name().toLowerCase().contains(osToCheck.toLowerCase()) && value.isSupported());
```

Reply:
@mrizzi can you explain why we need contains rather than equals here please? I don't understand this change.

Reply:
Well, for example an `osDescription` value could be `Red Hat Enterprise Linux Server release 7.4 (Maipo)` and another could be `Red Hat Enterprise Linux Server release 7.6` and both should match with `Red Hat Enterprise Linux`.
I know the `startsWith` would have worked as well but i see no enhancement in checking the it starts with since we're just interested in the content to appear somewhere.